assets/src/blocks/Columns/ColumnsBlock.js

```diff
@@ -8,11 +7,7 @@ const { registerBlockType } = wp.blocks;
 
 const getStyleLabel = (label, help) => {
   if (help) {
-    return (
-      <Tooltip text={help}>
-        <span>{label}</span>
-      </Tooltip>
-    );
+    return label . ' - ' . help;
```

Comment:
This notation doesn't work in JS, it crashes 🙃 I think you should write either 
```
label + ' - ' + help
```
or 
```
`${label} - ${help}`
```

Reply:
ops, my bad 😬 , I was just jump from PHP to JS very fast. Thanks 🙏